### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ieivanov**
Update readme to clarify `mantis` / `shrimpy` naming and document additional CLIs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +142,39 @@ mantis register \
...
+mantis estimate-stitching \
+    -i ./acq_name.zarr/*/*/* \
+    -o ./stitching.yml \
+    --channel DAPI
+    --percent-overlap 0.05
+# stitch fields of view
+mantis stitch \
+    -i ./acq_name.zarr/*/*/* \
+    -c  ./stitching.yml \
+    -o./acq_name_stitched.zarr/*/*/*
```
Comment (talonchandler) [inline README.md:162]:
```suggestion
    -c ./stitching.yml \
    -o ./acq_name_stitched.zarr/*/*/*
```

**Review by talonchandler — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -1,21 +1,23 @@
 # shrimPy: Smart High-throughput Robust Imaging & Measurement in Python
 ![acquisition and reconstruction schematic](docs/figure_3a.png)
 
-shrimPy (pronounced: ʃrɪm-pai) is a pythonic framework for high-throughput smart microscopy and high-performance analysis. The current alpha version of the framework is specific to the mantis microscope, described in our [preprint](https://www.biorxiv.org/content/10.1101/2023.12.19.572435v1), but extensible to a high throughput microscope that is controlled with [Micro-Manager](https://micro-manager.org/).
+shrimPy (pronounced: ʃrɪm-pai) is a pythonic framework for high-throughput smart microscopy and high-performance analysis. The current alpha version of the framework is specific to the mantis microscope, described in our [manuscript](https://www.biorxiv.org/content/10.1101/2023.12.19.572435v1), but extensible to a high throughput microscope that is controlled with [Micro-Manager](https://micro-manager.org/).
```
Comment (mattersoflight) [inline README.md:4]: @ieivanov now posted on PNAS Nexus webiste: https://academic.oup.com/pnasnexus/advance-article/doi/10.1093/pnasnexus/pgae323/

**Review by mattersoflight — APPROVED** [review]
The change looks good to me, the paper is online at https://academic.oup.com/pnasnexus/advance-article/doi/10.1093/pnasnexus/pgae323/

**Pull request merged**